src/nt_users_tool/read_config.py

```diff
@@ -0,0 +1,21 @@
+import yaml
+from constants import CONFIG_FILE_PATH, INPUT_FILE, OUTPUT_FILE
+
+def read_config_file() -> dict():
```

Comment:
⚠️ do not create a new dictionary here!
```suggestion
def read_config_file() -> dict:
```

If you want to be more precise, you can also specify the type of the dict's keys and values. For instance:
```Python
from typing import Dict, Any
def read_config_file() -> Dict[str, Any]:
```
(`dict[str, Any]` works with Python >= 3.10)

Reply:
Typo. I tried to apply the syntax you described but does not work with python 3.8.5 as you mentionned.

Reply:
Now fixed in 638ef90.